Classification/include/CGAL/Classification/ETHZ_random_forest_classifier.h

```diff
@@ -148,12 +148,17 @@ class ETHZ_random_forest_classifier
     std::vector<float> ft;
     
     for (std::size_t i = 0; i < ground_truth.size(); ++ i)
-      if (ground_truth[i] != -1)
+    {
+      int g = int(ground_truth[i]);
+      if (g != -1)
       {
         for (std::size_t f = 0; f < m_features.size(); ++ f)
           ft.push_back(m_features[f]->value(i));
-        gt.push_back(ground_truth[i]);
+        gt.push_back(g);
       }
+    }
+
+    std::cerr << "Using " << gt.size() << " inliers" << std::endl;
```

Comment:
May be better...?

CGAL_CLASSIFICATION_CERR << "Using " << gt.size() << " inliers" << std::endl;

It avoids messages when CGAL runs in silent mode.